HHS/Head-Start-TTADP
# [TTAHUB-1649] Include topics on TTA history/RTR topics & reasons graph

## Description of change
The TTA History tab of the recipient record page was unfortunately missed when we refactored the topics widget on the regional dashboard to include the topics from the AROs as well as the legacy field.

## How to test
View the RTR/TTA history page and ensure that the correct information is being loaded there.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1649


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
